src/main/java/cookbuddy/logic/commands/NewCommand.java

```diff
@@ -37,7 +37,7 @@
         + PREFIX_NAME + "Ham Sandwich "
         + PREFIX_INGREDIENTS + "bread, 2 slices; ham, 1 slice "
         + PREFIX_INSTRUCTIONS + "put ham between bread; eat sandwich "
-        + PREFIX_IMAGEFILEPATH + "C:\\Users\\zain_\\Downloads\\download.jpg "
+        + PREFIX_IMAGEFILEPATH + "path/to/image "
```

Comment:
I think it would be good if we show an example of a path here (so that a user knows how should they enter it) since the lines above are showing an example on how to use the new command to create a recipe.

Reply:
But then that's not platform-agnostic; we have at least 2 different OSes that handle stuff differently. Perhaps this could go in the UG instead?